.github/workflows/build.yaml

```diff
@@ -69,12 +104,13 @@ jobs:
             submodules: recursive
         - name: Download leap-dev.deb (Ubuntu 20 only)
           if: matrix.platform == 'ubuntu20'
-          uses: AntelopeIO/asset-artifact-download-action@v2
+          uses: AntelopeIO/asset-artifact-download-action@v3
           with:
             owner: AntelopeIO
             repo: leap
             file: 'leap-dev.*(x86_64|amd64).deb'
-            target: main
+            target: '${{needs.versions.outputs.leap-dev-target}}'
+            prereleases: ${{fromJSON(needs.versions.outputs.leap-dev-prerelease)}}
             artifact-name: leap-dev-ubuntu20-amd64
             container-package: experimental-binaries
             token: ${{github.token}}
```

Comment:
with `v3` you can get rid of this line

Reply:
Thanks for the reminder. I keep forgetting that is no longer required with `v3`.  Removed.